### ISSUE TITLE
No way to associate existing asset to an item

### ISSUE THREAD

**Issue by VanAxl** [pluginsGLPI/order#216]
Hi,

So I'm using GLPI for years now (currently in v9.3.2) and I just tried the plugin orders v2.2.1, which looks very useful to have a clean order management.

But I failed in some point accosting existing asset to my order.

Here is what I did:

I first create a _Product Reference_ , selecting my manufacturer in the list (Dell in my case) , with Item type _Computers_ , type _Laptop_, and last the Model (Latitude 7390).

Then I 've created my order. Adding one Latitude 7390 in the _Order Item_.I've validated the order, set the computer as delivered.

Now in the associated items, I want to link this computer to my existing asset. But the list is empty.
![image](https://user-images.githubusercontent.com/45239924/48856900-e9ac9580-edb7-11e8-9f4e-025645ecc99d.png)

My asset has the same type , model and Manufacturer as in the Orders.
![image](https://user-images.githubusercontent.com/45239924/48857015-37290280-edb8-11e8-96ea-93e0779b05e3.png)

I don 't see what is wrong.
Any advise ?

Thank you
VanAxl.

**Comment by xacobofg** [comment]
I tried to link it to an asset and I can
![image](https://user-images.githubusercontent.com/39123808/48857776-fdf19200-edb9-11e8-95d5-e8360eb39551.png)
Could it be an entity related issue?

**Comment by VanAxl** [comment]
I thought about entity issue as well. Because at first my order and the computer were not in the same. So I’ve moved both to the root entity. But that did not help.

**Comment by VanAxl** [comment]
I've started from scrash to create a new order, and I've also manually add an asset in Computer, all in Root entity.
So Here is the computer : 
![image](https://user-images.githubusercontent.com/45239924/48936342-ec6bcf80-ef0a-11e8-8a3b-729fac5dd7a9.png)

then my order : 
![image](https://user-images.githubusercontent.com/45239924/48936417-28069980-ef0b-11e8-936a-b5a4c5bb64b2.png)

My item in the order : 
![image](https://user-images.githubusercontent.com/45239924/48936435-3e145a00-ef0b-11e8-8651-9e7c9786c694.png)

Order is validated. Item is been taken to delivery.

But still no luck .
![image](https://user-images.githubusercontent.com/45239924/48936511-6dc36200-ef0b-11e8-954a-32387f2cbbe0.png)

I will try to have a look at the source code to see the sql query for this last page.

**Comment by VanAxl** [comment]
Ok, so after playing a lot with all option or settings, I've finally found what was wrong.
Not sure if it may tag as a bug, but here is what I found :

Whatever entity is set for my order, my item, or even the already present asset, the 'link to existing asset'  was not working if myself (the logged in user) was browsing the root entity (with tree structure). 

After sitting the view to an sub entity were I have computer, it displays results.

It might be better  If I can stay at the top of an entity where assets are present in different sub-entities, so that I can link every asset easily.

Anyway, now I have been able to link my first order to my 20 assets, playing with entities.
thanks.
